src/frontend/src/components/initialLogin/InitialLogin.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```

Comment:
Het component heet `InitialLogin`. Why? Initial log in klinkt als de eerste keer dat iemand inlogt. Misschien kun je het component beter renamen naar iets als `NotLoggedIn` of `RequestLogin`.